doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -7,14 +7,44 @@ Security
    :local:
    :depth: 3
 
-Nordic Matter samples leverage security features supported in the |NCS| that can be divided into three major categories:
+Nordic Matter samples leverage security features supported in the |NCS| that can be divided into four major categories:
 
+* Security features
 * Cryptography
 * Trusted storage
 * Securing production devices
 
 In the following sections you will learn more details about each listed category.
 
+Security features
+*****************
+
+In matter samples depending on the board, you can use some of the :ref:`security` features.
```

Comment:
```suggestion
Depending on the board, Matter samples can use a secure processing environment.
```
Leading with the board since the subheading also starts with it.